
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/pysnip/make/job.py", line 79, in run
    eval(pycode_compiled)
  File "snippets/50_lattices_exercises/50_lattices_exercises-JoinSemilattice.py", line 1, in <module>
    from act4e_tex import write_source_minted; from act4e_interfaces import JoinSemilattice; write_source_minted(JoinSemilattice, nodec=False,boxit=False,reduce_space=True)
  File "/Users/gioele/Desktop/act4e/book/ACT4E/ACT4E-private/src/act4e_tex/__init__.py", line 1, in <module>
    from .visualize import *
  File "/Users/gioele/Desktop/act4e/book/ACT4E/ACT4E-private/src/act4e_tex/visualize.py", line 7, in <module>
    from act4e_tests import alldata
  File "/Users/gioele/Desktop/act4e/book/ACT4E/ACT4E-private/src/act4e_tests/__init__.py", line 7, in <module>
    from .actual_tests import *
  File "/Users/gioele/Desktop/act4e/book/ACT4E/ACT4E-private/src/act4e_tests/actual_tests.py", line 6, in <module>
    from zuper_testint import ImplementationFail, TestContext, TestManagerInterface, TestRef, Tests
ModuleNotFoundError: No module named 'zuper_testint'
